src/main/java/com/example/Ingress_lab/dao/CustomerEntity.java

```diff
@@ -0,0 +1,16 @@
+package com.example.Ingress_lab.dao;
+
+import lombok.*;
+import org.springframework.stereotype.Component;
+
+@AllArgsConstructor
```

Comment:
just use:
@ NoArgConstructor
@ AllArgsConstructor
@ Data

[Optional]: use @ Builder for create object